src/com/AbstractFactory/OLEDB.java

```diff
@@ -0,0 +1,14 @@
+package com.AbstractFactory;
+
+public class OLEDB extends SQLFactory {
+	
+	static final String SQLSERVER = "SQLSEVER";
```

Comment:
This constant is not used, you are using Constant.SQLSERVER. Please remove it if not used.
Anyway is not common create a separate class for constants, constants are usually defined in the class it's used.